src/algolia/index.js

```diff
@@ -0,0 +1,62 @@
+import algoliasearch from 'algoliasearch';
+
+function createClient(appId, apiKey, indexName) {
+  if (!apiKey)
+    throw new Error(
+      'npm-search: Please provide the `apiKey` env variable and restart'
+    );
+
+  const client = algoliasearch(appId, apiKey);
+  return {
+    index: client.initIndex(indexName),
+    client,
+  };
+}
+
+/**
+ * Prepare algolia for indexing
+ * @param {object} config
+ */
+async function prepare(config) {
+  // Get main index and boostrap algolia client
+  const { index: mainIndex, client } = createClient(
+    config.appId,
+    config.apiKey,
+    config.indexName
+  );
+  const { index: bootstrapIndex } = createClient(
```

Comment:
looking at this code, it might make a lot more sense to simply call "algoliasearch()" twice now, and put the error here, instead of the wrapper. WDYT?

Reply:
I moved the error sooner, I'll let the wrapper for algoliasearch It's cleaner for var declaration